core/src/Cuda/Kokkos_Cuda_Parallel_MDRange.hpp

```diff
@@ -39,6 +39,34 @@
 namespace Kokkos {
 namespace Impl {
 
+template <typename ParallelType, typename Policy, typename LaunchBounds>
+int max_tile_size_product_helper(const Policy& pol, const LaunchBounds&) {
```

Comment:
The policy has the launch bounds info so we could deduce it
(Not asking you to change, it is already good enough)